tests/suites/appdata/task.sh

```diff
@@ -0,0 +1,21 @@
+test_appdata() {
+    if [ "$(skip 'test_appdata')" ]; then
+        echo "==> TEST SKIPPED: appdata tests"
+        return
+    fi
+
+    set_verbosity
+
+    echo "==> Checking for dependencies"
+    check_dependencies juju
+
+    file="${TEST_DIR}/test-appdata.txt"
+
+    bootstrap "test-appdata" "${file}"
+
+    test_appdata_int
+
+    destroy_model "test-appdata"
```

Comment:
Officially you shouldn't need this here.

Reply:
I added it because when running it with `BOOTSTRAP_REUSE_LOCAL` it would leave the test-appdata model around so the next run it would fail. That model isn't actually needed since `test_appdata_int` creates its own - is there a way to avoid creating test-appdata? Maybe I should add one?